Documentation/network/bgp.rst

```diff
@@ -6,9 +6,9 @@
 
 .. _bgp:
 
-**********
-BGP (beta)
-**********
+****************
+BGP (deprecated)
```

Comment:
nit, I'd probably consider making this a bit clearer by declaring it as MetalLB-based BGP Control Plane just to make it clearer how it differs from the new one. Feel free to follow up on that afterwards.

Reply:
follow up PR sent, check ya review queue :)